spec/system/dismiss_topics_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+describe "Filtering topics", type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:topic_list) { PageObjects::Components::TopicList.new }
+  let(:dismiss_new_modal) { PageObjects::Modals::DismissNew.new }
+  fab!(:group) { Fabricate(:group) }
+  fab!(:topic) { Fabricate(:topic) }
+
+  before do
+    user.groups << group
```

Comment:
```suggestion
  fab!(:user) { Fabricate(:user) }
  let(:topic_list) { PageObjects::Components::TopicList.new }
  let(:dismiss_new_modal) { PageObjects::Modals::DismissNew.new }
  fab!(:group) do 
    Fabricate(:group).tap { |g| g.add(user) }
  end
  fab!(:topic) { Fabricate(:topic) }

  before do
```

The user can also be added to the group within `fab!`.